sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -372,30 +351,28 @@ private void processCallback(final Log log, final CallbackProcessor callbackProc
                     public void run() {
                         if (log instanceof ManagedErrorLog) {
                             ManagedErrorLog errorLog = (ManagedErrorLog) log;
-                            if (errorLog.getFatal()) {
-                                final ErrorReport report = buildErrorReport(errorLog);
-                                UUID id = errorLog.getId();
-                                if (report != null) {
-
-                                    /* Clean up before calling callbacks if requested. */
-                                    if (callbackProcessor.shouldDeleteThrowable()) {
-                                        removeStoredThrowable(id);
-                                    }
-
-                                    /* Call back. */
-                                    HandlerUtils.runOnUiThread(new Runnable() {
+                            final ErrorReport report = buildErrorReport(errorLog);
+                            UUID id = errorLog.getId();
+                            if (report != null) {
 
-                                        @Override
-                                        public void run() {
-                                            callbackProcessor.onCallBack(report);
-                                        }
-                                    });
-                                } else {
-                                    MobileCenterLog.warn(LOG_TAG, "Cannot find crash report for the error log: " + id);
+                                /* Clean up before calling callbacks if requested. */
+                                if (callbackProcessor.shouldDeleteThrowable()) {
+                                    removeStoredThrowable(id);
                                 }
+
+                                /* Call back. */
+                                HandlerUtils.runOnUiThread(new Runnable() {
+
+                                    @Override
+                                    public void run() {
+                                        callbackProcessor.onCallBack(report);
+                                    }
+                                });
+                            } else {
+                                MobileCenterLog.warn(LOG_TAG, "Cannot find crash report for the error log: " + id);
                             }
                         } else {
-                            if (!(log instanceof ErrorAttachmentLog)) {
+                            if (!(log instanceof ErrorAttachmentLog) && !(log instanceof HandledErrorLog)) {
```

Comment:
You can probably do `else if { .. }` instead of `else { if { .. } }`.

Reply:
sure, it's existing code though